lib/features/children/overview/cubit/family_overview_state.dart

```diff
@@ -48,12 +48,12 @@ class FamilyOverviewUpdatedState extends FamilyOverviewState {
   List<Profile> sortedAdults(String givtAccountID) {
     return adults
       ..sort((a, b) {
-        final compareNames = a.firstName.compareTo(b.id);
+        final comapreId = a.id.compareTo(b.id);
```

Comment:
Just to be sure, why the comparing of IDs? As far as I understand the old code did:
Put whoever is the logged in user first, else compare the first names (so kinda sort alphabetically). The new code does: put whoever is the logged in user first, else order based on id (which to the user is an invisible number).

Reply:
sorry that shouldnt in there